ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -61,6 +61,8 @@ service PartyManagementService {
   // canton: completely different globally unique identifier is allocated.
   // Behind the scenes calls to an internal protocol are made. As that protocol
   // is richer than the surface protocol, the arguments take implicit values
+  // Party names are required to be non-empty US-ASCII strings built from letters, digits, space,
+  // colon, minus and underscore limited to 255 chars
```

Comment:
```suggestion
  // The party identifier suggestion must be a valid party name. Party names are required to be non-empty US-ASCII strings built from letters, digits, space,
  // colon, minus and underscore limited to 255 chars
```
This might not be obvious, given that the ledger is allowed to arbitrarily change the suggested identifier.

Reply:
Makes sense, thanks!